### PR TITLE
[E2E] Fixing broken steps in Allure report

### PR DESCRIPTION
**Pull request by ypolishchuk-ledger**
### ✅ Checklist

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

_Replace this text by a clear and concise description of what this pull request is about and why it is needed. Be sure to explain the problem you're addressing and the solution you're proposing._
_For libraries, you can add a code sample of how to use it._
_For bug fixes, you can explain the previous behaviour and how it was fixed._
_In case of visual features, please attach screenshots or video recordings to demonstrate the changes._

### ❓ Context

- **JIRA or GitHub link**: 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by VicAlbr — DISMISSED** [review]

File: apps/ledger-live-desktop/tests/page/abstractClasses.ts [hunk 1]
```diff
@@ -26,6 +27,7 @@ export abstract class Component extends PageHolder {
     return await this.page.waitForLoadState("load");
   }
 
+  @step("Wait for page to be fully loaded")
```
Comment (bharamboure-ledger) [inline apps/ledger-live-desktop/tests/page/abstractClasses.ts:30]: `wait for network calls to be done` might be more appropriate

File: apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts [hunk 2]
```diff
@@ -13,10 +13,11 @@ const e2eDelegationAccounts = [
     delegate: new Delegate(Account.SOL_1, "0.001", "Ledger by Figment"),
     xrayTicket: "B2CQA-2742",
   },
-  {
-    delegate: new Delegate(Account.NEAR_1, "0.01", "ledgerbyfigment.poolv1.near"),
-    xrayTicket: "B2CQA-2741",
-  },
+  //ToDo: enable when speculos interactions are refactored by @victor
```
Comment (bharamboure-ledger) [inline apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts:16]: maybe let's link a ticket ?

**Review by bharamboure-ledger — DISMISSED** [review]

File: apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts (hunk 2 above)
Comment (ypolishchuk-ledger) [reply apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts:16]: @VicAlbr can you please provide one?

File: apps/ledger-live-desktop/tests/page/abstractClasses.ts (hunk 1 above)
Comment (ypolishchuk-ledger) [reply apps/ledger-live-desktop/tests/page/abstractClasses.ts]: done

File: apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts (hunk 2 above)
Comment (VicAlbr) [reply apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts:17]: Yep, I'm doing the refacto on this ticket: QAA-306

**Review by bharamboure-ledger — DISMISSED** [review]

File: apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts (hunk 2 above)
Comment (ypolishchuk-ledger) [reply apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts]: thanks, ticket added to the comment

**Review by VicAlbr — APPROVED** [review]

**Pull request merged**